react/src/base/inputs/Textarea.stories.js

```diff
@@ -1,52 +1,104 @@
 import React from 'react';
 import SprkTextInput from './SprkTextInput/SprkTextInput';
+import SprkTextarea from './SprkTextarea/SprkTextarea';
+import SprkErrorContainer from './SprkErrorContainer/SprkErrorContainer';
+import SprkLabel from './SprkLabel/SprkLabel';
+import SprkHelperText from './SprkHelperText/SprkHelperText';
 import { markdownDocumentationLinkBuilder } from '../../../../storybook-utilities/markdownDocumentationLinkBuilder';
 
 export default {
   title: 'Components/Input/Textarea',
   decorators: [(story) => <div className="sprk-o-Box">{story()}</div>],
-  component: SprkTextInput,
+  component: SprkTextarea,
   parameters: {
-    jest: ['SprkErrorContainer', 'SprkInputIconCheck'],
+    jest: ['SprkTextarea'],
     info: `${markdownDocumentationLinkBuilder('input')}`,
   },
 };
 
-// Had to comment out because it breaks SB left uncommented
-// export const textarea = () => (
-
-// );
+export const textarea = () => (
+  <>
+    <SprkLabel htmlFor="sprk-textarea">Description</SprkLabel>
+    <SprkTextarea
+      name="textarea"
+      id="sprk-textarea"
+      additionalClasses="sprk-u-Width-100"
```

Comment:
Is there a reason we need `additionalClasses` on these stories?

Reply:
@bobwise, I should of left a comment on this one explaining haha. Right now, our inputs rely on that utility class that sets !important for the width. That is something we are going to change. We are going to add width: 100% to the input CSS classes so that the utility and the !important isn't needed anymore. Plan is to make those CSS changes with this release and then remove the `additionalClasses="sprk-u-Width-100"` from this story file. So, we're gonna go back in here in the next week or so and remove this line. It's there for now though.